Configuration/PyReleaseValidation/python/upgradeWorkflowComponents.py

```diff
@@ -2819,6 +2819,29 @@ def condition(self, fragment, stepList, key, hasHarvest):
     offset = 0.9001,
 )
 
+class UpgradeWorkflow_HeavyIon(UpgradeWorkflow):
+    def setup_(self, step, stepName, stepDict, k, properties):
+        stepDict[stepName][k] = merge([{'--procModifiers': 'phase2_pp_on_AA'}, stepDict[step][k]])
+        if 'GenSim' in step:
+            stepDict[stepName][k] = merge([{'--conditions': stepDict[step][k]["--conditions"].replace('_13TeV',''), '-n': 1}, stepDict[stepName][k]])
+        elif 'Digi' in step:
+            stepDict[stepName][k] = merge([{'-s': stepDict[step][k]["-s"].replace("DIGI:pdigi_valid","DIGI:pdigi_hi"), '--pileup': 'HiMixNoPU'}, stepDict[stepName][k]])
+    def condition(self, fragment, stepList, key, hasHarvest):
+        return fragment=='HydjetQMinBias_5519GeV' and '2026' in key and 'PU' not in key
+
+upgradeWFs['HeavyIon'] = UpgradeWorkflow_HeavyIon(
+    steps = [
+        'GenSimHLBeamSpot',
+        'DigiTrigger',
+        'RecoGlobal',
+        'HARVESTGlobal',
+        'ALCAPhase2'
+    ],
+    PU = [],
+    suffix = '_hi',
+    offset = 0.85,
```

Comment:
Please add this workflow to README.

Reply:
Done in d1b4049bd8dffaf205aa6e4f9c4851e06c50dc90